jest.config.base.cjs

```diff
@@ -26,6 +26,10 @@ module.exports = {
     ),
     // Handle monaco worker files
     '\\.worker.*$': 'identity-obj-proxy',
+    'react-transition-group': path.join(
+      __dirname,
+      './__mocks__/react-transition-group.js'
+    ),
```

Comment:
Yea if we don't have this here the mock doesn't get applied and all the tests fail. Guess it's how we have jest set up.
The automatic mocking happens for `__mocks__` folders next to the node_modules folders of our packages, but not at the root it would appear.
I'm not too invested in figuring it out beyond that, perhaps we switch to `vitest` at some point.